public/lessons/javascript/values.md

```diff
@@ -33,9 +47,89 @@ A **string** is an object that's a collection of characters, like a word or a se
 "Cherry Pie"
 ```
 
+# Slicing and Dicing
+
+Every string is made of lots of other strings.
+
+You can pull out parts of a string with the `slice` method.
+
+```js
+// this means "slice from character 0 to character 4"
+"blueberry".slice(0, 4) 
+
+// this means "slice from character 4 to the end
+"blueberry".slice(4)
+```
+
+These start and end numbers are called *indexes* (or *indices* if you're feeling fancy).
+
+[MDN: slice](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/slice)
+
+# String Indexing Explained
+
+Humans like to start counting at 1, but computers like to start counting at 0.
+
+This can be confusing, so here's a visualization to help explain it.
+
+Think of the indexes as pointing at the *spaces between* characters, as in this diagram:
+
+    | B | L | U | E | B | E | R | R | Y |
+    0   1   2   3   4   5   6   7   8   9
+     
+So with this picture in your mind, `slice`...
+  
+   * includes the character to the *right* of the start index
+   * includes the character to the *left* of the end index...
+   * ...but *excludes* the character to the *right* of the end index
+
+Try various start and end values in the console and see what happens!
+
+# Characters
+
+Q: A string is "a series of characters"... but what is a character?
+
+A: a character is a *number* (or *character code*) that stands for a *symbol*.
+
+|symbol|code|name|
+|---|---|---|
+| `A` | 65 | capital A |
+| `B` | 66 | capital B |
+| `Z` | 90 | capital Z |
+| `_` | 95 | underscore |
+| `a` | 97 | lowercase A |
+| ??? | 10 | newline |
+
+(Some characters stand for *unprintable* symbols like `newline` or `tab` or `bell`.)
+
+# Lab: Playing with Strings
+
+Open up a Node instance in your terminal, and let's play around with strings!
+
+    * Add two strings together that use the same type of quotes (e.g.`'Java' + 'Script'`)
+    * Add two strings together using different types of quotes (e.g. `"I'm double quoted!" + 'I am single quoted.')
+    * Subtract two strings (e.g. `'cats' - 'dogs'`)
+    * Add a string to a number
+
+What happens when you try to put quotes inside your string's quotes?
+
+# Lab: String Methods
+
+Strings have many operations attached to them. These attached operations are refered to as 'methods' (more on that later) and are quite useful. Let's try a few of them out in your console now:
+
+    * `"titanic".toUpperCase()`
+    * `"QUIETLY".toLowerCase()`
+    * `"Java".repeat(10)`
+    * `"banana".length`
+    * `"berry".charAt(1)`
+    * `"berry".charAt(0)`
+    * `"apple"[3]`
+    * `"banana".includes("nan")`
+    * `"banana".endsWith("ana")`
+    * `"blueberry".replace("blue", "black")`
```

Comment:
The back ticks on each of these is rendered with the markdown. Is this intentional?